test/operatorStage2.js

```diff
@@ -89,10 +93,34 @@ describe('Stage 2', async () => {
           expect(actualStandardBal).to.equal(expectedStandardBal);
         }
 
-        it('[final price (1.0)] rewards with TST successfully', async () => {
-          await testingSuite(etherBalances['125K'], rates.TWENTY_PC);
+        it('transfers TST rewards successfully when bonding with a custom rate', async () => {
+		  await OP2.connect(owner).addRate(rates.TWENTY_PC, 1);
+          await testingSuite(etherBalances['125K'], rates.TWENTY_PC, 1);
           await expectedTokBalance(125000, 1.2);
         });
+
+		it('transfers TST rewards successfully when bonding with the default rate', async () => {
+		  let twoPercent = 2000;
+		  await testingSuite(etherBalances['125K'], twoPercent, 52);
+		  await expectedTokBalance(125000, 1.02);
+		});
+
+		it('reverts when trying to bond with a non-added rate', async () => {
+		  let threePercent = 3000;
+		  let arbitraryWeeks = 36;
+		  await expect(testingSuite(etherBalances['125K'], threePercent, arbitraryWeeks)).to.be.revertedWith('err-missing-rate');
+		});
+
+		it('adds multiple new rates and grows the set of accepted rates', async() => {
+		  await OP2.connect(owner).addRate(rates.FIVE_PC, 10);
+		  await OP2.connect(owner).addRate(rates.TEN_PC, 20);
+		  await OP2.connect(owner).addRate(rates.TWENTY_PC, 40);
+
+		  let expectedRates = 4;
+		  let actualRates = (await OP2.showRates()).length;
+		  expect(actualRates).to.equal(expectedRates);
+		});
+
```

Comment:
i know you're going to say that it's messed up everywhere, but what's going on with your indenting here?

(i am a bit confused what you've been saying about tabs and spaces but that's a conversation for offline)

Reply:
yes, i'll conform it to what has been written before. we fix the styling once and for all after release.